src/Umbraco.Tests/Services/ContentServiceTests.cs

```diff
@@ -2474,6 +2474,234 @@ public void PublishingTest()
             */
         }
 
+        [Test]
+        public void Can_SaveAndRead_Names()
+        {
+            var languageService = ServiceContext.LocalizationService;
+
+            var langFr = new Language("fr-FR");
+            var langUk = new Language("en-UK");
+            var langDe = new Language("de-DE");
+            languageService.Save(langFr);
+            languageService.Save(langUk);
+            languageService.Save(langDe);
+
+            var contentTypeService = ServiceContext.ContentTypeService;
+
+            // fixme
```

Comment:
I think this needs to be explained to me :) I assume that these fixme's are part of later tasks and not to be completed for this one.

Reply:
Later task indeed.